modules/ossm-control-plane-deploy.adoc

```diff
@@ -114,29 +133,3 @@ The installation has finished successfully when the READY column is true.
 NAME           READY
 basic-install   True
```

Comment:
The output should have more than just these two columns. The READY column should show something like `7/7`, where the first number is the number of components that are ready, while the second is the total number of components. You may also want to explain that these numbers don't represent the number of Deployment/DaemonSet/StatefulSet resources, but the components that are listed in `SMCP.status.components`.